components/TodoCount.js

```diff
@@ -0,0 +1,17 @@
+class TodoCount {
+  constructor() {
+    this.$count = document.querySelector('.todo-count');
+    this.count = 0;
+  }
```

Comment:
```suggestion
  constructor(target) {
    this.$count = target;
    this.count = 0;
  }
```
이렇게 외부에서 주입해주는건 어떻게 생각하시나요?

컴포넌트에 메소드를 넘겨주는 것 처럼 DOM 객체도 넘겨줄 경우에 재활용 하기가 더 수월합니다!

Reply:
확실히 재활용 측면에서 의견 주신 부분이 낫다고 생각합니다! 감사합니다